SwiftCompilerSources/Sources/Optimizer/InstructionSimplification/SimplifyBuiltin.swift

```diff
@@ -162,6 +172,23 @@ private extension BuiltinInst {
     uses.replaceAll(with: literal, context)
     context.erase(instruction: self)
   }
+  
```

Comment:
Can you add tests for this in test/SILOptimizer/simplify_builtin.sil?

Reply:
Added!